### ISSUE TITLE
(low) "Pretty" is too subjective a word to help determine need for decriptive alt text

### ISSUE THREAD

**Issue by mpalmersg** [w3c/wai-easy-checks#170]
Location: 
*https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/image-alt/*

Current wording: Images that appear to be just to be ‘pretty’ should be marked with ‘Alt text = “”

Suggested revision: Images that appear to be 'decorative' only should be marked with ‘Alt text = “”

Rationale:

'Pretty' is too subjective a word. Something that has functional meaning can also be pretty.
